### PR TITLE
create and join new table owners

### PR DESCRIPTION
**Pull request by aradradev**
In this Pull Request, I've:
- Created new table owners
- Used JOIN clause to join the animal's table

[Screenshot](https://docs.google.com/document/d/1CcteUCE8WyM1txzlG5HW9ShcPacqXKHmem7IEmkocxs/edit?usp=sharing)

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -57,4 +57,39 @@ FROM animals
...
+FROM animals
+JOIN owners ON animals.owner_id = owners.id
+WHERE owners.full_name = 'Melody Pond';
+
+SELECT animals.name
+FROM animals
+JOIN species ON animals.species_id = species.id
+WHERE species.name = 'Pokemon';
+
+SELECT owners.full_name, animals.name
```
Comment (rloterh) [inline queries.sql:71]: - [ ] The creation of the table `owners` is not found in your queries, let's endeavor to add the same, like the code example below:

```sql
 CREATE TABLE owners (
   id int NOT NULL IDENTITY(1,1),
   full_name varchar(255),
   age int,
   PRIMARY KEY (id)
   );
```

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Abdoul, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 
🚦 _ 🏡 🏃‍♂️ 🏚️ 🌴 🗼

## Highlights 👍  
- Good job so far implementing update queries!
- ReadMe looks professional!

## Required Changes ♻️
_Kindly check the inline comments under the review._

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql [hunk 2]
```diff
@@ -11,3 +11,58 @@ vet_clinic(# );
...
+CREATE TABLE
+vet_clinic=# \d
+               List of relations
+ Schema |      Name      |   Type   |  Owner
+--------+----------------+----------+----------
+ public | animals        | table    | postgres
+ public | animals_id_seq | sequence | postgres
+ public | owners         | table    | postgres
+ public | owners_id_seq  | sequence | postgres
+(4 rows)
```
Comment (AbelG101) [inline schema.sql]: - [ ] [OPTIONAL] Please make sure to only copy proper SQL statements as I noticed you're copying the entire text from the `PSQL` CLI which contains unnecessary text that can't be run, so replace the above statement with the below SQL statement as referred by the previous reviewer
```sql
 CREATE TABLE owners (
   id int NOT NULL IDENTITY(1,1),
   full_name varchar(255),
   age int,
   PRIMARY KEY (id)
 );
```

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @aradradev 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Correct queries for each question ✔
- Correct use of Git Flow ✔

 ## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mukhammadrizooff — APPROVED** [review]
Hi @aradradev, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

 

## Highlights
- No linter errors ✔️ 
- Correct GitFlow ✔️ 
- Good implementations ✔️
- Descriptive `README` file ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**